tests/js/spec/views/organizationGroupDetails/actions.spec.tsx

```diff
@@ -110,12 +112,11 @@ describe('GroupActions', function () {
 
       const onReprocessEventFunc = jest.spyOn(ModalActions, 'openModal');
 
-      const wrapper = renderComponent(event);
-
-      const reprocessActionButton = wrapper.find('ReprocessAction');
-      expect(reprocessActionButton).toBeTruthy();
+      renderComponent(event);
+      const btn = await screen.findByLabelText('Reprocess this issue');
+      expect(btn).toBeInTheDocument();
 
-      reprocessActionButton.simulate('click');
+      userEvent.click(btn);
 
       await tick();
```

Comment:
i think this should be waitFor rather than enzyme tick

Reply:
oh oops, why did I leave this here